report/src/main/webapp/app/services/history.service.js

```diff
@@ -0,0 +1,136 @@
+/*
+  * AET
+  *
+  * Copyright (C) 2013 Cognifide Limited
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+define(['angularAMD', 'endpointConfiguration'], function (angularAMD) {
+  'use strict';
+  angularAMD.factory('historyService', historyService);
+
+  /**
+   * Service responsible for fetching suite's history
+   */
+  function historyService($rootScope, $http, endpointConfiguration) {
+    var suiteHeaders;
+    $rootScope.endpointUrl = endpointConfiguration.getEndpoint().getUrl;
```

Comment:
Is that necessary to bind this value to the `$rootScope` variable?
Maybe you can use local variable instead?

Reply:
This value isn't used anywhere in the service file - it is used only in the view file so it has to be put in the `$rootScope`, otherwise it wouldn't be possible to use it in the `historyModal.view.html` file.

Reply:
Where exactly in `historyModal.view.html` is used `endpointUrl`?